amazon-kinesis-client-multilang/src/main/java/software/amazon/kinesis/coordinator/KinesisClientLibConfiguration.java

```diff
@@ -1098,6 +1104,7 @@ public KinesisClientLibConfiguration withMaxRecords(int maxRecords) {
      */
     public KinesisClientLibConfiguration withIdleTimeBetweenReadsInMillis(long idleTimeBetweenReadsInMillis) {
         checkIsValuePositive("IdleTimeBetweenReadsInMillis", idleTimeBetweenReadsInMillis);
+        idleTimeBetweenReadsInMillis = Math.max(idleTimeBetweenReadsInMillis, MIN_IDLE_MILLIS_BETWEEN_READS);
```

Comment:
Not sure if this is necessary, as I believe this is only used in the KCL1, but I would like to verify this. Similar change was done [here](https://github.com/awslabs/amazon-kinesis-client/pull/1270/files#diff-f8708ff806f9fe8125c4e291f763635f9c5963854219f90fcc0f513dd7181026R980).